### ISSUE TITLE
Megamenu is not operable by VoiceOver

### ISSUE THREAD

**Issue by ewillebrands** [primefaces/primevue#7220]
The menu-items of submenu's in the megamenu are not reachable with VoiceOver, and probably other screenreaders.
Aria-haspopup is used correctly, however there is no indication that a submenu is expanded. Hence VoiceOver is not able to enter the submenu. Setting aria-expanded to true/false depending on the state should probably fix that.

**Comment by tugcekucukoglu** [comment]
<img width="1129" alt="Image" src="https://github.com/user-attachments/assets/e4d709cf-32d5-41e0-8334-cba2e13fb9d1" />

We can not replicate the issue. It seems to be that `aria-expended` is implemented.

**Comment by tugcekucukoglu** [comment]
Closing due to inactivity.
